graphql-test-utils/graphql-models/user.js

```diff
@@ -21,6 +21,7 @@ const User = new Factory()
   .attr('launchdarklyTrackingTenant', 'gcp-eu')
   .attr('defaultProjectKey', () => null)
   .attr('businessRole', () => faker.name.jobDescriptor())
+  .attr('idTokenUserInfo', () => null)
```

Comment:
I was having this kind of error in the tests:
![image](https://user-images.githubusercontent.com/97907068/193602829-bcfdce41-5bce-435c-96a6-70280c467ae7.png)

The info that I could find about this error suggest the `in-memory` apollo cache needs to write all the optional properties of the object it wants to store in the cache ([reference](https://stackoverflow.com/a/50167010))